### PR TITLE
Feature/oidf 46

### PR DESCRIPTION
**Pull request by maikel-maas**

### CODE REVIEW

File: modules/admin-server/src/main/resources/db/migration/V0.1.0__Account.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+CREATE TYPE "public"."KMS_TYPE" AS ENUM ('LOCAL');
```
Comment (sanderPostma) [inline modules/admin-server/src/main/resources/db/migration/V0.1.0__Account.sql:1]: we should not hardcode schema "public". It's up to the config file to set the schema name

File: modules/admin-server/src/main/resources/db/migration/V0.1.0__Account.sql [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+CREATE TABLE "public"."account" (
```
Comment (sanderPostma) [inline modules/admin-server/src/main/resources/db/migration/V0.1.0__Account.sql:3]: we should not hardcode schema "public". It's up to the config file to set the schema name

**Review by sanderPostma — CHANGES_REQUESTED** [review]

File: modules/admin-server/src/main/resources/db/migration/V0.1.0__Account.sql (hunk 1 above)
Comment (maikel-maas) [reply modules/admin-server/src/main/resources/db/migration/V0.1.0__Account.sql]: Removed public from the enum name

File: modules/admin-server/src/main/resources/db/migration/V0.1.0__Account.sql (hunk 2 above)
Comment (maikel-maas) [reply modules/admin-server/src/main/resources/db/migration/V0.1.0__Account.sql]: Removed public from the table name

**Review by sanderPostma — APPROVED** [review]
LGTM

**Pull request merged**